cdap-examples/Purchase/src/test/java/co/cask/cdap/examples/purchase/PurchaseAppTest.java

```diff
@@ -62,7 +63,8 @@ public void test() throws TimeoutException, InterruptedException, IOException {
 
     try {
       // Wait for the last Flowlet processing 5 events, or at most 5 seconds
-      RuntimeMetrics metrics = RuntimeStats.getFlowletMetrics("PurchaseHistory", "PurchaseFlow", "collector");
+      RuntimeMetrics metrics = RuntimeStats.getFlowletMetrics(Constants.DEFAULT_NAMESPACE, "PurchaseHistory",
```

Comment:
Ok I am not sure if it is a good idea to require users to use Constants.DEFAULT_NAMESPACE without introducing the concepts of Namespaces in 2.7.